src/utils/set.ts

```diff
@@ -24,17 +28,38 @@ module Plottable {
         var index = this._values.indexOf(value);
         if (index !== -1) {
           this._values.splice(index, 1);
+          this._setSize(this._values.length);
           return true;
         }
         return false;
       }
 
+      private _setSize(size: number) {
```

Comment:
Will `_setSize` ever be used for anything other than setting to `this._values.length`?  If not, we can remove the parameter here and just use `this._values.length` inside the code